common/services/prismic/link-resolver.ts

```diff
@@ -45,14 +45,17 @@ function linkResolver(doc: Props | DataProps): string {
 
   if (
     type === 'exhibition-guides' ||
-    type === 'exhibition-texts' ||
     type === 'exhibition-highlight-tours' ||
     type === 'exhibition-guides-links' ||
     type === 'Exhibition text' ||
     type === 'Exhibition highlight tour'
   )
     return `/guides/exhibitions/${uid}`;
 
+  if (type === 'exhibition-texts') {
+    return `/guides/exhibitions/${uid}/captions-and-transcripts`;
+  }
+
   if (type === 'visual-stories') {
```

Comment:
We're not returning the "correct" URL here, since we don't have 'data', it goes to the fallback URL - do we mind? (maybe one for Lauren).

http://localhost:3000/visual-stories/hard-graft 
should be
http://localhost:3000/exhibitions/hard-graft-work-health-and-rights/visual-stories

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/issues/11543